app/assets/javascripts/discourse/app/components/sidebar/sidebar-section-form-link.js

```diff
@@ -0,0 +1,62 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+
+export default class SidebarSectionFormLink extends Component {
+  @tracked dragCssClass;
+
+  constructor() {
+    super(...arguments);
+    this.link = this.args.link;
```

Comment:
I don't think this line is needed since we can just refer to it as `this.args.link`?